packages/react-spinbutton/jest.config.js

```diff
@@ -0,0 +1,10 @@
+const { createConfig, resolveMergeStylesSerializer } = require('@uifabric/build/jest/jest-resources');
+const path = require('path');
+
+const config = createConfig({
```

Comment:
You're missing config/tests.js and config/api-extractor.json (both of these can be copied from another project, though I'm about to update the API extractor config--so you might need to wait on that)